src/emissions-workspace/emissions.service.ts

```diff
@@ -28,6 +30,7 @@ export class EmissionsWorkspaceService {
   async export(params: EmissionsParamsDTO): Promise<EmissionsDTO> {
     const promises = [];
     const DAILY_TEST_SUMMARIES = 0;
+    const MATS_MONITOR_HRLY_VALUES = 1;
```

Comment:
Don't need this index since it will use the hourly operating index and the data for mats monitor hourly values will be added in the hourly operating service